src/main/kotlin/com/vanniktech/maven/publish/nexus/CloseAndReleaseRepositoryTask.kt

```diff
@@ -1,10 +1,12 @@
 package com.vanniktech.maven.publish.nexus
 
 import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.TaskAction
 
 open class CloseAndReleaseRepositoryTask : DefaultTask() {
 
+  @Input
   lateinit var nexusOptions: NexusOptions
```

Comment:
Can these be inputs if they're not serializable?

Reply:
Missed that. I changed it to Nested and annotated NexusOptions